wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -723,6 +725,49 @@ func ScreenOpenCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (s
 	return update, nil
 }
 
+func ScreenReorderCommand(ctx context.Context, pk *scpacket.FeCommandPacketType) (sstore.UpdatePacket, error) {
```

Comment:
something weird happened to your indentation here.  are you running gofmt?

Reply:
gofmt on save is not working on my end. not sure why. will just run manually.